src/checkstyle/checks.xml

```diff
@@ -56,8 +56,8 @@
         <property name="message" value="Math.toIntExact should only be used with static imports" />
     </module>
     <module name="RegexpSingleline">
-        <property name="format" value="^import static .*\.of;$" />
-        <property name="message" value="Methods named &quot;of&quot; should never be statically imported" />
+        <property name="format" value="^import static .*\.(of|copyOf|valueOf);$" />
+        <property name="message" value="Methods named &quot;of&quot;, &quot;copyOf&quot; and &quot;valueOf&quot; should never be statically imported" />
```

Comment:
This should technically be "or" instead of "and"

Reply:
Or how about we change this to
```
The following methods may not be statically imported: of, copyof, valueOf

Reply:
Or just `This method cannot be statically imported` (the error marker will tell everything that needs to be told).